Slim/Plugin/WiMP/ProtocolHandler.pm

```diff
@@ -253,15 +252,6 @@ sub _gotTrack {
 	} else {
 		$params->{successCb}->();
 	}	
-
-	# trigger playback statistics update
-	if ( $info->{duration} > 2) {
-		# we're asked to report back if a track has been played halfway through
-		my $params = {
-			duration => $info->{duration} / 2,
-			url      => $params->{url},
-		};
-	}
```

Comment:
Sure enough I missed this change... what happened here? Why did you remove this part of the code?

Reply:
euh ... because you asked me :-)

Reply:
Oh, because it's unused code? Oh my...

Reply:
Yes, and I always do what the Boss says

Reply:
And about memory, I think I'm in the same boat, just rougher waters... That's why I was begging for the PR to be reviewed and accepted so that when arrows flows, at least it's fresh in my head. It was pretty difficult to go back for the volatile part already :-)